home.html

```diff
@@ -12,8 +12,28 @@
 </head>
 
 <body>
+
+    <!----header-->
```

Comment:
The index.html page is looking good.  One thing you can do it make the header and nav the same size with the rest of the content page like you did with the about.html.